lib/src/Uncertainty/Algorithm/MetaModel/LinearModel/LinearModelAnalysis.cxx

```diff
@@ -316,24 +315,34 @@ Scalar LinearModelAnalysis::getFisherPValue() const
   return FisherSnedecor(dofModel, dof).computeComplementaryCDF(FStatistic);
 }
 
-/* Kolmogorov-Smirnov normality test */
+/* Estimator of the standard deviation */
+Scalar LinearModelAnalysis::getResidualsStandardError() const
+{
+  const Scalar sigma2 = linearModelResult_.getSampleResiduals().computeRawMoment(2)[0];
+  const UnsignedInteger dof = linearModelResult_.getDegreesOfFreedom();
+  const UnsignedInteger n = linearModelResult_.getSampleResiduals().getSize();
+  // TODO: The formula sigma2 * sampleSize / dof is highly suspiscious: this is already unbiased in LinearModelAlgorithm
+  const Scalar stdError = std::sqrt(sigma2 * n / dof);
```

Comment:
```suggestion
  const Scalar stdError = std::sqrt(sigma2 * (n-1) / dof);
```

To be checked, but that might be the correct formula

Reply:
According to [^Bingham2010] page 67, the unbiased estimator of the variance is:

$$
\widehat{\sigma}^2 = \frac{n - p}{SSE}
$$

where $SSE$ is the sum of squared errors, $n$ is the sample size and $p$ is the number of parameters to estimate. The standard error of the regression is the root mean squared error. 

[^Bingham2010]: Bingham, N. H., & Fry, J. M. (2010). Regression: Linear models in statistics. Springer Science & Business Media.